crates/pallet-transaction-fees/src/lib.rs

```diff
@@ -45,30 +50,13 @@ struct CollectedFees<Balance: Codec> {
     tips: Balance,
 }
 
-#[derive(Encode, Decode, TypeInfo)]
-struct BlockTransactionByteFee<Balance: Codec> {
-    // The value of `transaction_byte_fee` for the current block
-    current: Balance,
-    // The value of `transaction_byte_fee` for the next block
-    next: Balance,
-}
-
-impl<Balance: Codec + tokens::Balance> Default for BlockTransactionByteFee<Balance> {
-    fn default() -> Self {
-        BlockTransactionByteFee {
-            current: Balance::max_value(),
-            next: Balance::max_value(),
-        }
-    }
-}
-
 #[frame_support::pallet]
 mod pallet {
-    use super::{BalanceOf, BlockTransactionByteFee, CollectedFees, WeightInfo};
+    use super::{BalanceOf, CollectedFees, WeightInfo};
     use frame_support::pallet_prelude::*;
     use frame_support::traits::Currency;
     use frame_system::pallet_prelude::*;
-    use subspace_runtime_primitives::FindBlockRewardAddress;
```

Comment:
Why is this refactoring necessary? It was an implementation detail of `pallet-transaction-fees`, you shouldn't need to access it from the outside. I also don't understand how it helps you if the rest of APIs that worked with this type remained private.